src/parsers/manifestjson.js

```diff
@@ -100,6 +100,8 @@ export default class ManifestJSONParser extends JSONParser {
 
   checkKeySupport(support, minVersion, key, isPermission = false) {
     if (support.firefox) {
+      // We don't have to support gaps in the `@mdn/browser-compat-data`
+      // information for Firefox Desktop.
```

Comment:
Well, this is not strictly correct but I don't know if we are checking all props in the manifest.